### PR TITLE
Create inception_transferlearning.ipynb

### PR DESCRIPTION
**Pull request by ashizhao**
performs transfer learning on inception v1 neural network to classify images of grocery store food

### CODE REVIEW

File: notebooks/neural_networks/inception_transferlearning.ipynb [hunk 1]
```diff
@@ -0,0 +1,1107 @@
...
+    "preds = predictions.take(num_preds)\n",
+    "for idx in range(num_preds):\n",
+    "    true_label = str(map_to_label(map_groundtruth_label(truth[idx].label.to_ndarray())))\n",
+    "    pred_label = str(map_to_label(map_predict_label(preds[idx])))\n",
+    "    print idx + 1, ')', 'Ground Truth label: ', true_label\n",
+    "    print idx + 1, ')', 'Predicted label: ', pred_label\n",
+    "    print \"correct\" if true_label == pred_label else \"wrong\""
+   ]
+  },
+  {
```
Comment (yangw1234) [inline notebooks/neural_networks/inception_transferlearning.ipynb:982]: Maybe we should plot the Accuracy-Epoch curve on the validation set to the performance of this classifier
